pages/services/ember-consulting.vue

```diff
@@ -1,86 +1,216 @@
           >
-            Ship Shape knows what they're doing. A thoughtful approach, pristine
-            code and a 200% effort in everything they do, makes recommending
-            them a no-brainer.
-          </h2>
+            <nuxt-img
+              class="h-20 inline-block mb-8 w-auto"
+              src="/svgs/icons/chart.svg"
+              alt=""
+            />
+            <p class="text-white">Scalable UI architecture</p>
+          </div>
 
-          <p
-            class="font-bold"
-            itemprop="author"
-            itemscope
-            itemtype="http://schema.org/Person"
+          <div
+            class="bg-navy flex flex-col justify-items-center pt-12 lg:px-16"
+            data-aos="flip-down"
+            data-aos-delay="400"
           >
-            <span itemprop="name">James Dixon</span>,
-            <span itemprop="jobTitle"> Founder </span>
-            of
-            <a href="https://www.scoutforpets.com/"> Scout </a>
+            <nuxt-img
+              class="h-20 inline-block mb-8 w-auto"
+              src="/svgs/icons/retainer.svg"
+              alt=""
+            />
+            <p class="text-white">Simple updates</p>
+          </div>
+
+          <div
+            class="bg-navy flex flex-col justify-items-center pt-12 lg:px-16"
+            data-aos="flip-down"
+            data-aos-delay="600"
+          >
+            <nuxt-img
+              class="h-20 inline-block mb-8 w-auto"
+              src="/svgs/icons/sail.svg"
+              alt=""
+            />
+            <p class="text-white">Commitment to stability</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="section flex flex-wrap justify-center !pt-40"
+      data-aos="fade-down"
+    >
+      <div
+        class="
+          section-content
+          gap-20
+          grid grid-cols-1
+          items-center
+          lg:gap-32 lg:grid-cols-2
+        "
+      >
+        <div>
+          <nuxt-img
+            class="h-auto m-auto max-w-2xl w-full"
+            format="webp"
+            height="772"
+            src="/img/hero-images/mentoring.png"
+            width="1312"
+            alt=""
+          />
+        </div>
+        <div>
+          <h2>We’ll never leave you high and dry</h2>
+          <p>
+            Ship Shape is committed to leaving every engineering team we work
+            with better than we found it. Not only do we build robust, reliable
+            software, but we also train your staff along the way to understand,
+            maintain, and improve your product after our engagement ends.
           </p>
         </div>
       </div>
     </div>
 
-    <WhyEmber />
+    <SingleQuote :testimonial="testimonial" />
 
     <CommunityInvolvement />
 
-    <RecentBlogPosts :posts="posts" />
+    <WaveMoreLinks class="md:px-20" />
   </div>
 </template>
 
 <script>
 import { generateMeta } from '~/utils/meta';
 
+const testimonial = {
+  name: 'Namey Nameson',
```

Comment:
@hannakim91 for this testimonial, can you please update it to be the one from Roger from Expel we use on the homepage?